src/components/application_manager/test/commands/mobile/show_constant_tbt_request_test.cc

```diff
@@ -209,6 +230,27 @@ TEST_F(ShowConstantTBTRequestTest, Run_NextTurnIcon_Canceled) {
   command->Run();
 }
 
+TEST_F(ShowConstantTBTRequestTest, Run_NextTurnIcon_WrongSyntax) {
+  MessageSharedPtr msg = CreateMsgParams();
+  SmartObject next_turn_icon(smart_objects::SmartType_Map);
+  next_turn_icon[strings::value] = "Wrong Syntax\\n";
+  (*msg)[strings::msg_params][strings::next_turn_icon] = next_turn_icon;
+
+  SharedPtr<ShowConstantTBTRequest> command(
+      CreateCommand<ShowConstantTBTRequest>(msg));
+
+  EXPECT_CALL(mock_app_manager_, application(kConnectionKey))
+      .WillOnce(Return(mock_app_));
+  EXPECT_CALL(mock_app_manager_, ManageMobileCommand(_, _));
+  EXPECT_CALL(mock_app_manager_, GetPolicyHandler()).Times(0);
+  EXPECT_CALL(mock_message_helper_, ProcessSoftButtons(_, _, _, _)).Times(0);
+  EXPECT_CALL(*mock_app_, app_id()).Times(0);
+
+  EXPECT_CALL(*mock_app_, set_tbt_show_command(_)).Times(0);
+
```

Comment:
@dcherniev I see dublicate code here and above maybe you need move common part to separate method